<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAPR</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="SceneWidget" name="srcFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QScrollArea" name="sidebarScrollArea">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="SidebarWidget" name="sidebarWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>68</width>
          <height>519</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="project">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="importAction"/>
    <addaction name="exportAction"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <widget class="QMenu" name="supportsMenu">
     <property name="title">
      <string>Опоры</string>
     </property>
     <addaction name="leftSuppAction"/>
     <addaction name="rightSuppAction"/>
    </widget>
    <addaction name="newCoreAction"/>
    <addaction name="supportsMenu"/>
   </widget>
   <addaction name="project"/>
   <addaction name="edit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newCoreAction">
   <property name="text">
    <string>Добавить стержень</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="leftSuppAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Левая</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="rightSuppAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Правая</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="importAction">
   <property name="text">
    <string>Импорт...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>Экспорт...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SidebarWidget</class>
   <extends>QWidget</extends>
   <header>sidebarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SceneWidget</class>
   <extends>QFrame</extends>
   <header>scenewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
